packages/front-end/components/Settings/AthenaForm.tsx

```diff
@@ -86,6 +68,68 @@ const AthenaForm: FC<{
           </div>
         </>
       )}
+      {(isCloud() || params.authType == "assumeRole") && (
```

Comment:
I think this should be `!isCloud() && ` instead of `isCloud() || `

Reply:
Good catch, will fix that

Reply:
fixed